entrypoint.js

```diff
@@ -4,9 +4,23 @@ shim("check-master");
 const {Toolkit} = require('actions-toolkit');
 
 Toolkit.run(async tools => {
-  const path = tools.arguments.path || '.*';
+  let paths = [];
+
+  switch(typeof tools.arguments.path) {
+    case 'string':
+      paths = [ tools.arguments.path ]
+      break;
+    case 'array':
+      paths = tools.arguments.path
+      break;
+    default:
+    case 'undefined':
+      paths = [ '.*' ]
+      break;
+  }
+
```

Comment:
```suggestion
let path = tools.arguments.path || '.*';
```

Reply:
We no longer use `path`, don't think this is required. Have changed this area of the code slightly, let me know what you think.